### DISCUSSION TITLE
点歌自监督经典工作

### DISCUSSION THREAD

**Discussion by tfwang08** [mli/paper-reading#4] [General]
之前Alexnet里也讲到，很多大佬认为网络应该是自监督学习的，可否讲一下自监督的经典工作。
我不是这个领域的，希望有看到的大佬可以补充下经典论文。

**Comment by Albert-Ma** [comment]
小白也只大概了解近几年的一些比较好的自监督论文：
NLP： **word2vec**, UlmFit, ELMO, **GPT**, **BERT**
CV:  **SimCLR**, **BYOL**, **MOCO**
留个大佬补充一些更早的工作可

**Reply by fengshi-cherish** [reply]
补充audio的wav2vec，vq-wav2vec。还有之前的CPC，我记得林达华老师之前有几篇工作是做这块相关的。

**Reply by absolutemind** [reply]
视觉上还有深度估计 (depth estimation),双目深度估计和基于视频的单目深度估计。音频任务上还有音频分割（speech separation）
